OSDOCS-10591 OCP 4.13.42 RNs (#76267)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[id="ocp-4-13-41-updating"]
==== Updating
To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

[id="ocp-4-13-42"]
=== RHSA-2024:2875 - {product-title} 4.13.42 bug fix update

Issued: 2024-05-23

{product-title} release 4.13.42, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2875[RHSA-2024:2875] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2877[RHSA-2024:2877] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.13.42 --pullspecs
----

[id="ocp-4-13-42-bug-fixes"]
==== Bug fixes

> * Previously, for the `cluster:capacity_cpu_cores:sum` metric, nodes with the`infra` role but not `master` role were not assigned a value of `infra` for the `label_node_role_kubernetes_io` label. With this update, nodes with the `infra` role but not `master` role are now correctly labeled as `infra` for this metric.(link:https://issues.redhat.com/browse/OCPBUGS-33581[*OCPBUGS-33581*])

Reviewer: Anyway to refer to "master" as something else (to abide by conscious language efforts)? 
If not, totally understand, just wanted to mention it.

Author: Checking with the dev.

Author: Looks like this is the language we have to use for now. The style guide says it's okay as long as it's not paired with 'slave.'

> * Previously, the load balancing algorithm did not differentiate between active and inactive services when determining weights, and it employed the `random` algorithm excessively in environments with many inactive services or environments routing backends with weight 0. This led to increased memory usage and a higher risk of excessive memory consumption. With this release, changes are made to optimize traffic direction towards active services only and prevent unnecessary use of the `random` algorithm with higher weights, reducing the potential for excessive memory consumption. (link:https://issues.redhat.com/browse/OCPBUGS-33448[*OCPBUGS-33448*])

Reviewer: Suggestion as a way to remove passive voice..

* Previously, timeout values larger than what Golang could parse were not properly validated. Consequently, timeout values larger than what HAProxy could parse caused issues with HAProxy. With this update, if the timeout specifies a values larger than what can be parsed, it is capped at the maximum that HAProxy can parse. As a result, issues are no longer caused for HAProxy. (link:https://issues.redhat.com/browse/OCPBUGS-33280[*OCPBUGS-33280*])

> * Previously, a kernel regression that was introduced on {product-title} 4.14.14 caused kernel issues, such as nodes crashing and rebooting, in nodes that mounted to CephFS storage. In this release, the regression issue is fixed so that the kernel regression issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-33252[*OCPBUGS-33252*])

Reviewer: Suggestion to remove clause.

* Previously, some container processes created by using the `exec` command persisted even when CRI-O stopped the container. Consequently, lingering processes led to tracking issues, causing process leaks and defunct statuses. With this release, CRI-O tracks the `exec` calls processed for a container and ensures that the processes created as part of the `exec` calls are terminated when the container is stopped.(link:https://issues.redhat.com/browse/OCPBUGS-33174[*OCPBUGS-33174*])

> * Previously, Terraform would create the compute server group with the policy set for the control plane. As a consequence, the 'serverGroupPolicy' property of the install-config.yaml file was ignored for the compute server group. With this release, the server group policy set in the install-config.yaml file for the compute MachinePool is correctly applied at install-time in the Terraform flow. (link:https://issues.redhat.com/browse/OCPBUGS-33062[*OCPBUGS-33062*])

Reviewer: //server group difference
installation?
```suggestion
* Previously, Terraform would create the compute server group with the policy set for the control plane. As a consequence, the 'serverGroupPolicy' property of the install-config.yaml file was ignored for the compute server group. With this release, the server group policy set in the install-config.yaml file for the compute MachinePool is correctly applied during installation in the Terraform flow. (link:https://issues.redhat.com/browse/OCPBUGS-33062[*OCPBUGS-33062*])
```

Reviewer: Question: Are these referring to the same thing? 
"compute server group with the policy set" and "the server group policy set"
If so, should they be identified in the same way? Hopfully what I am asking makes sense :)

Author: Checking with dev

Author: Made a change for clarity.
